Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp

```diff
@@ -270,7 +281,11 @@ void APlayCppSdkActor::SaveClient(FString &output, bool &success,
 
     String sessioninfo = _coreClient->save_client();
     output = UTF8_TO_TCHAR(sessioninfo.c_str());
-    success = true;
+    success = FFileHelper::SaveStringToFile(
+        output, *(FPaths::ProjectDir() + "sessioninfo.json"),
```

Comment:
cannot save to projectdir by unreal coding paradigm.

FString path = FPaths::ProjectConfigDir() + filename;
use config folder or other folder

Reply:
For ProjectConfigDir, it would save to Config folder.
damonchen@HKMAC0741 ~/Documents/play-unreal-demo/simple (git)-[feat/ios_support] % find . -type f -name "sessioninfo.json"
`./Config/sessioninfo.json`

Reply:
In unreal, there are other folders, for example, ProjectSavedDir, ProjectUserDir etc, I guess, seems `ProjectSavedDir` is more appropriate? Session info is one kind of saved data.

Reply:
I can not find a so-called safe folder for android, it seems access folders outside the ProjectDir is not preferred. I'll double check.

Reply:
https://docs.unrealengine.com/4.27/en-US/Basics/DirectoryStructure/
> Contains files that are generated by the engine such as config files and logs. These files can be deleted and rebuilt.

Saved dir is the best option for saving sessioninfo.json.